nengo/builder/operator.py

```diff
@@ -201,6 +201,42 @@ def step():
         return step
 
 
+class SlicedCopy(Operator):
+    """Copy from `a` to `b` with slicing: `b[b_slice] = a[a_slice]`"""
+    def __init__(self, a, b, a_slice=Ellipsis, b_slice=Ellipsis,
+                 kind='set', tag=None):
+        assert kind in ['set', 'inc', 'update']
```

Comment:
Yeah, I put that in just because we could use such an op in the future. But I can remove it if people don't want it.

Also, re asserts: for things in the Builder, where the user should never get an error if we test things properly, asserts are fine IMO. This is just a sanity check for developers.